docs/admin-manual/cm-configuration.rst

```diff
@@ -244,27 +244,27 @@ entries` section for definitions of these configuration variables.
     accounting group, with a defined group quota, then this attribute is
     the integer number of slots currently utilized by the group.
 
-:index:`RemoteGroup<single: RemoteGroup; ClassAd attribute, ephemeral>`\ ``RemoteGroup``
+:index:`RemoteGroup<single: RemoteGroup; ClassAd attribute, ephemeral>`\ :ad-attr:`RemoteGroup`
     The accounting group name of the owner of the currently running job.
 
 :index:`RemoteGroupQuota<single: RemoteGroupQuota; ClassAd attribute, ephemeral>`\ ``RemoteGroupQuota``
```

Comment:
```suggestion
:index:`RemoteGroupQuota<single: RemoteGroupQuota; ClassAd attribute, ephemeral>`\ :ad-attr:`RemoteGroupQuota`
```

Reply:
I want to think about where to put these.  `RemoteGroupQuota` only shows up in the negotiator's copy of the job ad, so I'm worried that defining in this job ad attribute section would give people the idea that it is available everywhere.